.github/workflows/build.yml

```diff
@@ -153,7 +153,7 @@ jobs:
       - name: lint
         run: |
           ./scripts/mk_all.sh
-          lean --run scripts/lint_mathlib.lean
+          lean --run scripts/lint_mathlib.lean --github
```

Comment:
BTW, you changed the wrong file here.  There's a big warning on top:
```bash
# DO NOT EDIT THIS FILE!!!

# This file is automatically generated by mk_build_yml.sh
# Edit build.yml.in instead and run mk_build_yml.sh to update.
```

Reply:
Thanks for catching this, I've seen this warning before but totally missed it this time. I'll fix this tonight or tomorrow if nobody beats me to it

Reply:
@alexjbest it looks like it's fixed in #11645, which I'll put on the queue shortly

Reply:
Great! Do either of you have an easy to implement way to prevent this sort of thing happening again?
E.g. a check in CI that the workflow files are the same as those generated from the .in file? How hard would that be to check?

Reply:
That shouldn't be too hard.  I believe a `./.github/workflows/mk_build_yml.sh && git diff --exit-code .github` should do the trick.